utilities/flink/flink_compaction_filter.h

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+#include <atomic>
+#include <string>
+#include "rocksdb/compaction_filter.h"
+#include "rocksdb/slice.h"
+#include <chrono>
+#include <functional>
+#include <utility>
+#include <include/rocksdb/env.h>
+
+namespace rocksdb {
+namespace flink {
+
+#define BITS_PER_BYTE static_cast<size_t>(8)
```

Comment:
Why do you prefer defines over static constants?

Reply:
I will try to convert them into static constants